capsules/src/touch.rs

```diff
@@ -41,6 +43,9 @@ impl Default for App {
             events_buffer: None,
             ack: true,
             dropped_events: 0,
+            x: 0,
+            y: 0,
+            status: 3,
```

Comment:
Could status be changed to be a `TouchStatus` type, and an `Unstarted` variant added to `TouchStatus`? I find this initialization to `3` confusing and prone to breakage in the future if additional variants are added to `TouchStatus`.

Reply:
Done, thank you for the tip.